src/pytypes/onefuzztypes/enums.py

```diff
@@ -281,6 +281,15 @@ def available(cls) -> List["ScalesetState"]:
         unavailable = [cls.shutdown, cls.halt, cls.creation_failed]
         return [x for x in cls if x not in unavailable]
 
+    @classmethod
+    def is_resizing(cls) -> List["ScalesetState"]:
+        """ set of states that indicate scaleset is resizing """
```

Comment:
nit: neither halt or shutdown is 'resizing'.  Can a different verb be used here?